Tasks/DownloadPipelineArtifactV1/task.loc.json

```diff
@@ -9,7 +9,7 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 1
+    "Patch": 0
```

Comment:
Why bump the patch down?

Reply:
Hi @stephenmichaelf , The original version should be 0. We had increased the patch , just for testing purpose.